lib/core_extensions.rb

```diff
@@ -123,6 +123,10 @@ def self.audited(*args)
     super
     self.auditing_enabled = false if Foreman.in_setup_db_rake?
   end
+
+  def self.supports_authorize?
```

Comment:
This is a good idea by @ohadlevy - this way we can control/override what has authorization.
I just forgot to write this as a comment in the code but in general if you have a resource that has nothing to do with `delete/create/...` you override this method to return `false`.

@timogoebel  WDYT?

Reply:
I like the idea, however I would not make this global. I think it's enough to check if it includes the `Authorizable` module, see 
```
irb(main):007:0> Model.include?(Authorizable)
=> true
irb(main):008:0> Hash.include?(Authorizable)
=> false
```